### ISSUE TITLE
Do not restrict creating only one criteria (Vocabulary) on a field

### ISSUE THREAD

**Issue by kazoompa** [obiba/mica2#3945]
In Mica Admin Client, the taxonomy editor do not permit creating more than one vocabulary based on one field. There are cases where having multiple vocabularies on the same fiels is useful. For example, having `continents` and `countries` both on the same `countryIso` field.

**Comment by ymarcon** [comment]
And in order to avoid confusion, add an attribute `subset=true` so that in the search UI the "any" and "none" are not selected ("in" and "not in" are the only accurate requests).

**Comment by kazoompa** [comment]
Regarding `subset`, didn't the Client admin worked without this flag?

**Comment by ymarcon** [comment]
No, `any` and `none` must be hidden because it covers all the possible values of the field, and not just the ones that are enumerated by the vocabulary.

**Comment by kazoompa** [comment]
Will require Criterion widget UI modifications. Would also be nice to handle non-binary AND/OR operators since the back-end can already handle that.
